### DISCUSSION TITLE
Is a Splitter Node similar to vuejs blocks possible in vue-flow?

### DISCUSSION THREAD

**Discussion by kvedula18** [bcakmakoglu/vue-flow#1042] [Q&A]
First of all, great library. While I can use this library, our requirement is to have splitters based on values of the node, and a capability to add connectors to a Node based on the values of the Node. A simple one is a splitter with Yes/No values. More detailed ones could be based on the status of an entity. Is the Splitter kind of node doable using vue-flow?

https://github.com/ghostiam/vue-blocks

**Comment by bcakmakoglu** [comment]
You can implement custom nodes with whatever functionality you want and with whatever number of handles (which you call *connectors* I believe).

[Check out the custom node example to get an idea.](https://vueflow.dev/examples/nodes/)
